### PR TITLE
hotfix/TR-5000/pick-proper-commits

### PR DESCRIPTION
**Pull request by shaveko**
related to https://oat-sa.atlassian.net/browse/TR-5000

### Description

The https://github.com/oat-sa/extension-tao-itemqti/tree/v29.6.9 version which v29.6.10 was based on did not contain the required strings, so this hotfix is based on https://github.com/oat-sa/extension-tao-itemqti/tree/v29.6.9 with all required changes in place.

### How to test

Check with local installation and Japaneese interface:

 - Choice interaction with `allowEliminate` should contain label which is positioned horisontally
 - Extended text interaction with max characters constraint should have `%d characters maximum` translated

### CODE REVIEW

File: views/scss/qti/_choice.scss [hunk 1]
```diff
@@ -44,6 +44,7 @@
             left: 50%;
             transform: translateX(-50%);
             bottom: calc(100% + 20px);
+            white-space: nowrap;
```
Comment (gabrielfs7) [inline views/scss/qti/_choice.scss:47]: This is a backporting for https://github.com/oat-sa/extension-tao-itemqti/pull/2296/files#diff-22f4c217e33cba23248709f515f82a69be5f0d48af8046b39656e938c9729270R47 correct? 

I see the PR I've linked we did not change `views/css/item-creator.css.map`. Is that ok? Is it fixing the missing translations reported by @AntonNeviarouski ?

Comment (jsconan) [reply views/scss/qti/_choice.scss:47]: The map file is only there for metadata when debugging, it helps map the code from the bundle to the source code. Being there or not will not impact the feature.

**Review by jsconan — APPROVED** [review]
It works as expected
- before:
  image
- after:
  image

**Review Checklist**
- [ ] ~~New code is covered by tests (if applicable)~~
- [ ] ~~Tests are running successfully (old and new ones) on my local machine (if applicable)~~
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] ~~New code is not subject to concurrency issues (if applicable)~~ 
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Review by jsconan — COMMENTED** [review]
wait, the second statement regarding the translation is not true:
> Extended text interaction with max characters constraint should have %d characters maximum translated

image

**Review by jsconan — APPROVED** [review]
The constraint for the characters count is now translated
image

**Pull request merged**